### PR TITLE
Broken images tst page

### PR DESCRIPTION
**Pull request by anacosantos**

### CODE REVIEW

File: specs/basic-auth/basic-auth.spec.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-    })
+  beforeAll(() => {
+    autheticationPage = new BasicAuth()
+  })
 
-})
\ No newline at end of file
+  it('Shows up the authentication', () => {
+    // The Solution to Authentication Pop Up:driver.get(protocol://Usename:Password@URL Address);
+    browser.driver.get('https://admin:admin@the-internet.herokuapp.com/basic_auth')
```
Comment (mvgmenezes) [inline specs/basic-auth/basic-auth.spec.js:13]: put this URL in a CONST on top of this class

File: specs/broken-images/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+## Title exist
+- Access Url
+- Verify if title Broken Images exist on the web page
```
Comment (mvgmenezes) [inline specs/broken-images/README.md:3]: exists

**Review by mvgmenezes — CHANGES_REQUESTED** [review]

File: specs/basic-auth/basic-auth.spec.js (hunk 1 above)
Comment (anacosantos) [reply specs/basic-auth/basic-auth.spec.js:13]: done

File: specs/broken-images/README.md (hunk 2 above)
Comment (anacosantos) [reply specs/broken-images/README.md:3]: done

**Review by mvgmenezes — APPROVED** [review]

**Pull request merged**